video2numpy/read_vids_cv2.py

```diff
@@ -6,21 +6,38 @@
 from .resizer import Resizer
 from .shared_queue import SharedQueue
 from .utils import handle_youtube
+from .utils import extract_audio_from_url
 
 
-def read_vids(vids, worker_id, take_every_nth, resize_size, batch_size, queue_export):
+def read_vids(vids, 
+             worker_id, 
+             take_every_nth, 
+             resize_size, 
+             batch_size, 
+             queue_export_video,
+             queue_export_audio,
+             output_dir,
+             no_audio,
+             no_video):
     """
-    Reads list of videos, saves frames to Shared Queue
+    Reads list of videos, saves frames to Shared Queue,
+    or optionally, save audios to Shared Queue. 
 
     Input:
       vids - list of videos (either path or youtube link)
       worker_id - unique ID of worker
       take_every_nth - offset between frames of video (to lower FPS)
       resize_size - new pixel height and width of resized frame
       batch_size - max length of frame sequence to put on shared_queue (-1 = no max).
-      queue_export - SharedQueue export used re-create SharedQueue object in worker
+      queue_export_video  - SharedQueue export used re-create SharedQueue object in worker
```

Comment:
add information that video queue export is for video frames and audio is for audio

Reply:
Ok, I have added now, however it is possible that the sentence after my modification becomes not readable, because I do not understant the meaning of "SharedQueue export used re-create SharedQueue object in worker"